docs/docker_compose.md

```diff
@@ -53,9 +60,106 @@ When you declare either `docker_build` or `fast_build`,
 Tilt will find the image name in your `docker-compose.yml`,
 and use its own container-updating strategy instead of the one in the `docker-compose.yml` file.
 
+## Sample Project With `docker_build`
+
+Let's look at a simple example app that runs Redis and a NodeJS-based server with docker-compose.
```

Comment:
nit: `docker-compose` is the command. the tool is called `Docker Compose`.

Reply:
good point!